pkg/server/plugin/keymanager/awskms/awskms.go

```diff
@@ -364,11 +357,22 @@ func (p *Plugin) fetchAliasesPage(ctx context.Context, marker *string) (*string,
 	p.log.Debug("Found aliases", "num_aliases", len(aliasesResp.Aliases))
 
 	for _, alias := range aliasesResp.Aliases {
-		if alias.AliasName == nil || alias.TargetKeyId == nil {
+		if alias.AliasName == nil {
```

Comment:
Isn't an alias name a required property for an `Alias`? Should we add a warn log here in the weird off-chance that we didn't get an alias name back in the response to indicate there might be some issue with the AWS API?

Reply:
yeap, a warning o debug at least

Reply:
Thanks! I will add a warning. The docs never mention a situation where this could be nil. I only added this check because it's a possibility.